index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="assets/css/reset.css">
+    <link href="https://fonts.googleapis.com/css?family=Hanalei+Fill|New+Rocker|Pirata+One" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="assets/css/style.css">
+    
+    
+    <title>Document</title>
+</head>
+<body background="https://cdn.blizzardwatch.com/wp-content/uploads/2016/12/wowscrnshot092410142903.jpg">
```

Comment:
You can add background into your css file and edit it so it would stretch to the page. Currently if the browser is bigger than the site, it would just make tiles of the background image